Twig/VersionExtension.php

```diff
@@ -4,12 +4,12 @@
 
 use Shivas\VersioningBundle\Service\VersionManager;
 use Twig_Extension;
-use Twig_Extension_GlobalsInterface;
+use Twig\Extension\GlobalsInterface;
```

Comment:
This is unrelated to the `open_basedir` issue and should be addressed in another PR.